### ISSUE TITLE
./install: line 125: -u: command not found

### ISSUE THREAD

**Issue by vincentux** [YunoHost-Apps/rocketchat_ynh#75]
Fail install,  

```
Système d’exploitation Debian 9.9 64bit (Linux 4.9.0-9-amd64)
yunohost 3.5.2.2 (stable)
yunohost-admin 3.5.2 (stable)
moulinette 3.5.2 (stable)
ssowat 3.5.2.1 (stable) 
```

```
2019-06-02 19:07:07,484: DEBUG - ++ tr '[:upper:]' '[:lower:]'
2019-06-02 19:07:07,484: DEBUG - ++ echo tar.gz
2019-06-02 19:07:07,484: DEBUG - + src_format=tar.gz
2019-06-02 19:07:07,484: DEBUG - + src_extract=true
2019-06-02 19:07:07,484: DEBUG - + '[' rocket.chat-0.73.2.tgz = '' ']'
2019-06-02 19:07:07,484: DEBUG - + local local_src=/opt/yunohost-apps-src/rocketchat/rocket.chat-0.73.2.tgz
2019-06-02 19:07:07,484: DEBUG - + test -e /opt/yunohost-apps-src/rocketchat/rocket.chat-0.73.2.tgz
2019-06-02 19:07:07,484: DEBUG - ++ wget -nv -O rocket.chat-0.73.2.tgz https://releases.rocket.chat/0.73.2/download
2019-06-02 19:07:21,917: DEBUG - + local 'out=2019-06-02 19:07:21 URL:https://cdn-download.rocket.chat/build/rocket.chat-0.73.2.tgz?bda9f17422ca7b2e124645bf35cd4440-19 [157101187/157101187] -> "rocket.chat-0.73.2.tgz" [1]'
2019-06-02 19:07:21,918: DEBUG - + sha256sum -c --status
2019-06-02 19:07:21,918: DEBUG - + echo '3dc3eb11f383f7b72b0f23fedb305b6a566fa536a1e5087a4398255deeb864d8 rocket.chat-0.73.2.tgz'
2019-06-02 19:07:23,221: DEBUG - + mkdir -p /var/lib/rocketchat
2019-06-02 19:07:23,221: DEBUG - + true
2019-06-02 19:07:23,221: DEBUG - + '[' tar.gz = zip ']'
2019-06-02 19:07:23,221: DEBUG - + local strip=
2019-06-02 19:07:23,221: DEBUG - + '[' true '!=' false ']'
2019-06-02 19:07:23,221: DEBUG - + '[' true == true ']'
2019-06-02 19:07:23,221: DEBUG - + local sub_dirs=1
2019-06-02 19:07:23,221: DEBUG - + strip='--strip-components 1'
2019-06-02 19:07:23,221: DEBUG - + [[ tar.gz =~ ^tar.gz|tar.bz2|tar.xz$ ]]
2019-06-02 19:07:23,221: DEBUG - + tar -xf rocket.chat-0.73.2.tgz -C /var/lib/rocketchat --strip-components 1
2019-06-02 19:07:54,400: DEBUG - ++ wc -l
2019-06-02 19:07:54,412: DEBUG - ++ find /var/cache/yunohost/from_file/rocketchat_ynh-master/scripts/../sources/patches/ -type f -name 'app-*.patch'
2019-06-02 19:07:54,412: DEBUG - + ((  0 > 0  ))
2019-06-02 19:07:54,412: DEBUG - + test -e /var/cache/yunohost/from_file/rocketchat_ynh-master/scripts/../sources/extra_files/app
2019-06-02 19:07:54,413: DEBUG - + chown -R rocketchat: /var/lib/rocketchat
2019-06-02 19:07:57,319: DEBUG - + cd /var/lib/rocketchat/programs/server/
2019-06-02 19:07:57,320: WARNING - ./install: line 125: -u: command not found
2019-06-02 19:07:57,321: DEBUG - + -u rocketchat npm install --production
2019-06-02 19:07:57,322: WARNING - !!
2019-06-02 19:07:57,322: DEBUG - + ynh_exit_properly
2019-06-02 19:07:57,323: WARNING -   rocketchat's script has encountered an error. Its execution was cancelled.
2019-06-02 19:07:57,323: DEBUG - + local exit_code=127
2019-06-02 19:07:57,323: WARNING - !!
2019-06-02 19:07:57,324: DEBUG - + '[' 127 -eq 0 ']'
2019-06-02 19:07:57,324: WARNING - 
2019-06-02 19:07:57,324: DEBUG - + trap '' EXIT
2019-06-02 19:07:57,325: DEBUG - + set +eu
2019-06-02 19:07:57,325: DEBUG - + echo -e '!!\n  rocketchat'\''s script has encountered an error. Its execution was cancelled.\n!!'
2019-06-02 19:07:57,325: DEBUG - + type -t ynh_clean_setup
2019-06-02 19:07:57,325: DEBUG - + ynh_die
2019-06-02 19:07:57,325: DEBUG - + local legacy_args=mc
2019-06-02 19:07:57,325: DEBUG - + args_array=([m]=message= [c]=ret_code=)
2019-06-02 19:07:57,326: DEBUG - + declare -Ar args_array
2019-06-02 19:07:57,326: DEBUG - + local message
2019-06-02 19:07:57,326: DEBUG - + local ret_code
2019-06-02 19:07:57,326: DEBUG - + ynh_handle_getopts_args
2019-06-02 19:07:57,326: DEBUG - + set +x
2019-06-02 19:07:57,326: DEBUG - + echo ''
2019-06-02 19:07:57,326: DEBUG - + exit 1
```

**Comment by rosareven** [comment]
I got the same error.

**Comment by rosareven** [comment]
@vincentux I created a fork that fixed line 125 - it just missed sudo. You can install from my fork in the mean time.

**Comment by vincentux** [comment]
Thank you,
I don't have time to look right now, but as soon as I can I test this ;)

**Comment by selamanse** [comment]
fixed by merging #76

**Comment by selamanse** [comment]
please feel free to reopen if this issue still exists
